backend/src/backend/models.py

```diff
@@ -148,7 +148,11 @@ class Meta(BaseMeta):
     title: Optional[Title] = ormar.ForeignKey(Title, related_name="books")
 
     async def book_name(self) -> str:
-        return (await self.metadata.get(name="Name")).value
+        try:
+            return (await self.metadata.get(name="Name")).value
+        except ormar.exceptions.NoMatch:
+            # if the book doesn't have metadata Name
+            return ""
```

Comment:
Great but we should return `None`

Reply:
@rgaudin we're using following schema for this
```py
class BooksListSendSchema(BaseModel):
    id: uuid.UUID
    name: str
```
If I return `None` it will fail the validation of `name: str`

Reply:
A book cannot not have a Name. So the proper behavior is probably to let the ormar exception raise (or raise a custom InvalidBook Exception).
Haven't checked but I believe our add book endpoint prevents creating Book without a Name. If not, we should.

Reply:
@rgaudin 
> I believe our add book endpoint prevents creating Book without a Name. If not, we should.
In the schema of receiving Book data, we have the `metadata` dict as mandatory
```py
class BookAddSchema(BaseModel):
    id: uuid.UUID
    period: datetime.date
    counter: Dict[str, Any]
    article_count: int
    media_count: int
    size: int
    metadata: Dict[str, Any]
    url: pydantic.HttpUrl
    zimcheck: Dict[str, Any]
```
But this doesn't check if the `Name` key is present in it.
We're able to create Book without that. We should probably add another layer of validation?

Reply:
Yep, I'll create an issue